core/src/main/java/org/elasticsearch/index/analysis/WordDelimiterGraphTokenFilterFactory.java

```diff
@@ -0,0 +1,101 @@
+/*
```

Comment:
I can do that. I added it because the test that checks whether all analysis components are exposed failed otherwise.